### DISCUSSION TITLE
This site can't be reached in window 11 msi

### DISCUSSION THREAD

**Discussion by ShaungBhone** [leokhoa/laragon#724] [Q&A]
<img width="960" alt="image" src="https://github.com/leokhoa/laragon/assets/56265471/20f1f12f-e4b9-409e-a794-06612ccb1aec">

I got the issue with pretty url.

**Comment by ghost** [comment]
It may occur due to the virtual Hosts. Please check your virtual host configuration file to ensure it is correctly configured.
